tests/foreman/api/test_hostcollection.py

```diff
@@ -448,22 +454,25 @@ def test_positive_add_subscription(self):
                     auth=settings.server.get_credentials(),
                     headers={'content-type': 'application/json'},
                 )
-                assert prod_name in req.text, 'Subscription not applied HC members'
-
-    @tier1
-    @pytest.mark.stubbed
-    def test_positive_remove_subscription(self):
-        """Try to remove a subscription from a host collection
-
-        :id: fdf43e57-5101-4270-9750-afe26f77c53c
-
-        :steps:
-
-            1. Create a new or use an existing subscription
-            2. Add the subscription to the host collection
-            3. Remove the subscription from the host collection
-
-        :expectedresults: The subscription was added to the host collection
-
-        :CaseImportance: Critical
-        """
+                assert prod_name in req.text, 'Subscription not applied to HC members'
+            # Remove the subscription
+            # Call nailgun to make the API PUT to members of Host Collection
+            entities.Host().bulk_remove_subscriptions(
+                data={
+                    "organization_id": self.org.id,
+                    "included": {"ids": host_ids},
+                    "subscriptions": [{"id": product_subscription.id, "quantity": 1}],
+                }
+            )
+            # GET the subscriptions from hosts and assert they are gone
+            for host_id in host_ids:
+                req = request(
+                    'GET',
+                    '{0}/api/v2/hosts/{1}/subscriptions'.format(
+                        settings.server.get_url(), host_id
+                    ),
+                    verify=False,
+                    auth=settings.server.get_credentials(),
+                    headers={'content-type': 'application/json'},
+                )
```

Comment:
I vaguely feel we had this conversation before, but that really should be part of Nailgun.

Reply:
Yes, we have:
https://github.com/SatelliteQE/nailgun/issues/750